.github/workflows/ci-pull-requests.yml

```diff
@@ -61,6 +61,9 @@ jobs:
       - name: Install dependencies
         run: pnpm install --frozen-lockfile
 
+      - name: Configure Turbo cache
```

Comment:
This only configures the cache for the `lint` task. don't we want to set this up for the unit tests as well?